docs/feature_pointing_device.md

```diff
@@ -136,6 +149,16 @@ Also see the `POINTING_DEVICE_TASK_THROTTLE_MS`, which defaults to 10ms when usi
 
 **`POINTING_DEVICE_GESTURES_CURSOR_GLIDE_ENABLE`** is not specific to Cirque trackpad; any pointing device with a lift/contact status can integrate this gesture into its driver. e.g. PMW3360 can use Lift_Stat from Motion register. Note that `POINTING_DEVICE_MOTION_PIN` cannot be used with this feature; continuous polling of `pointing_device_get_report()` is needed to generate glide reports.
 
+#### Relative mode gestures
+
+| Gesture Setting                        | Description                                                                                                                      | Default       |
+| -------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `CIRQUE_PINNACLE_SIDE_SCROLL_ENABLE`   | (Optional) Enable side scroll. Touch originating on right side can trigger vertical scroll (IntelliSense trackpad style).        | _not defined_ |
+| `CIRQUE_PINNACLE_TAP_ENABLE`           | (Optional) Enable tap to "left click". Works on both sides of a split keyboard.                                                  | _not defined_ |
+| `CIRQUE_PINNACLE_SECONDARY_TAP_ENABLE` | (Optional) Tap in upper right corner of the trackpad will result in "right click". `CIRQUE_PINNACLE_TAP_ENABLE` must be enabled. | _not defined_ |
```

Comment:
Do side scroll and secondary tap work on circular trackpads? I wasn't able to trigger these.

Reply:
Yes, they should both work. Secondary tap is a bit awkward, since you need to exactly hit the "sweet spot"